### ISSUE TITLE
Markdown preview doesn't reopen after closing

### ISSUE THREAD

**Issue by maexlich** [onivim/oni#1873]
Using the current release of Oni (0.3.1) on macOS high Sierra, the markdown preview does not reopen after closing it once.
As soon as it was hidden using a keybinding, it is not possible to make it be displayed again without restarting Oni.

The following error message appears in the dev console:

```
bundle.js:1 Received notification for: TypeError: Cannot read property 'innerSplit' of null
Please report this error.
t.error @ bundle.js:1
0.bundle.js:241 Uncaught TypeError: Cannot read property 'innerSplit' of null
    at t.find (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:213:104938)
    at Array.find (native)
    at p._getAugmentedWindowSplitFromSplit (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:213:104927)
    at p.createSplit (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:213:103955)
    at MarkdownPreviewEditor.open (/Applications/Oni.app/Contents/Resources/app/extensions/oni-plugin-markdown-preview/lib/index.js:162:45)
    at MarkdownPreviewEditor.toggle (/Applications/Oni.app/Contents/Resources/app/extensions/oni-plugin-markdown-preview/lib/index.js:155:18)
    at Command.oni.commands.registerCommand.Command [as execute] (/Applications/Oni.app/Contents/Resources/app/extensions/oni-plugin-markdown-preview/lib/index.js:192:17)
    at i.executeCommand (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:1:933)
    at s.handleKey (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:1:5623)
    at _onRootKeyDown (file:///Applications/Oni.app/Contents/Resources/app/lib/browser/0.bundle.js:241:261975)
bundle.js:1 Received notification for: TypeError: Cannot read property 'innerSplit' of null
Please report this error.
```

It seems that some reference is deleted when the markdown window is closed, leading to problems when trying to reopen it.

**Comment by TalAmuyal** [comment]
I think it is resolved in `master`, getting a new build to verify.

**Comment by TalAmuyal** [comment]
Verified - it is solved on `master` but present in `v0.3.1`.
If you want to try `master`, please note that now the configurations are searched for in `~/.config/oni/config.js` rather than the old path.
I solved it using `ln -s` so both `v0.3.1` and `master` can find the configs.

Closing this issue, please reopen if you think it's needed or if the issue persists.
